.github/workflows/helm.yml

```
@@ -0,0 +1,48 @@
+name: Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
```

Nesse caso faz sentido mantermos o mesmo padrão do `cubos/cli`?

```suggestion
  release:
    types: [published]
```

---

É uma boa prática